.github/workflows/sanity.yml

```diff
@@ -22,7 +22,7 @@ jobs:
       with:
         go-version: ${{ env.GO_VERSION }}
     - name: Install golint
-      run: go get -u golang.org/x/lint/golint
+      run: go install golang.org/x/lint/golint@latest
```

Comment:
@elezar I was not able to get the CI checks to pass without this. I believe this is needed as a `go get` alters the go mod state